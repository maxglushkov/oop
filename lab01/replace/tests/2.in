абвгдеёж
гдеёжёжёжзийк
ёж
ЁжёжёЖ
1 ёж, 2 ёж.