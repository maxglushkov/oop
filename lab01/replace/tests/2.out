абвгдеёжёж
гдеёжёжёжёжёжёжзийк
ёжёж
ЁжёжёжёЖ
1 ёжёж, 2 ёжёж.